projects/react/package.json

```diff
@@ -29,9 +29,17 @@
         "Nikita Barsukov <nikita.s.barsukov@gmail.com>",
         "Georgiy Lunin <nextzeddicus@gmail.com>"
     ],
+    "devDependencies": {
+        "@testing-library/react": "12.1.5",
+        "@testing-library/react-hooks": "8.0.1",
+        "@testing-library/user-event": "^14.4.3",
+        "@types/react": ">=17.0.0",
+        "@types/react-dom": ">=17.0.0",
+        "react-test-renderer": ">=17.0.0"
+    },
     "peerDependencies": {
         "@maskito/core": "^0.11.0",
-        "react": ">=16.8",
-        "react-dom": ">=16.8"
+        "react": ">=17.0.0",
+        "react-dom": ">=17.0.0"
```

Comment:
why?

Reply:
@nsbarsukov I think we should keep the lowest possible version for greater compatibility.

Reply:
Yep, it is just an attempt to fix [new error](https://github.com/Tinkoff/maskito/actions/runs/4938046549/jobs/8827331876?pr=293).
I will revert it when i will find solution to it.

Reply:
fixed